### ISSUE TITLE
Maryland DL version "01" has invalid file type in header

### ISSUE THREAD

**Issue by General-Fault** [c0shea/IdParser#24]
The "01" version of the Maryland driver license is not compliant with the AMVAA standard.  According to the published standard, the 5th field in the PDF417 data should always be the characters "ANSI ". I have a Maryland DL for which the 5th field is "AMVAA". This is the error on which IdParser complains  and stops parsing. 

> System.ArgumentException: The file type is invalid. Expected "ANSI "
> Parameter name: input
>    at IdParser.IdParser.ValidateFormat(String input)
> 
~~There are also several other compliance issues in this barcode.~~

~~- The header appears to be missing field 8 (number of entries), or field 7 (jurisdiction version number), or the fields are one byte instead of two, impossible to tell which as they could both be "01", "0" or "1".~~
~~- There is an extra "DL" prepended to the subfile data.~~
~~- The full customer name is merged into an undefined comma separated "DAA" field rather than split into "DCS", "DAC", "DAD" fields.~~
~~- There are a few other undefined fields present, specifically "DAR", "DAS", "DAT" and "DBH".~~
~~- And just for fun, the dates are in the canadian format "YYYYMMDD" rather than the US format "MMDDYYYY".~~

The following sample barcode data has been sanitized  - 

`@<LF><RS><CR>AAMVA9999990101DL00290175DLDAQT-520-098-009-843<LF>DAADOE,SUE,,<LF>DAG9999 FAKE DR<LF>DAIANYTOWN<LF>DAJMD<LF>DAK11111      <LF>DARC   <LF>DAS          <LF>DAT     <LF>DAU999<LF>DAW999<LF>DBA20211102<LF>DBB19720211<LF>DBC9<LF>DBD20131024<LF>DBHY<CR>`
~~

**Comment by c0shea** [comment]
Does it parse anything if you set the validation level to none?

It's definitely the 2000 standard, as I found the undefined fields in the PDF. I may have never added support for those since they were removed in favor of new codes in the 2003 and later standards.

![image](https://user-images.githubusercontent.com/17714002/40455900-e5643f9e-5ebc-11e8-8645-f5da686cebba.png)

Regarding the name, that was how it was formatted in 2000, where it's LastName, FirstName, MiddleInitial. However, the fun is that not all 3 are always specified, the delimiters may still be present for all, and the delimiter isn't always a comma. I'm pretty sure I handled those cases in the code. The dates are also screwy in 2000.

**Comment by General-Fault** [comment]
Wow... should have checked the 2000 spec. Id didn't occur to me that the 2016 spec document wouldn't call out the outdated entries. My mistake. I'll edit the issue to remove the information about the header inconsistencies except for the unexpected "AMVAA" file type.  

The support may or may not be there. I was listing differences from the spec as I saw them, but the parser stopped at the file type.  I'll test it with validation disabled. I had to go back and check my own source logs, but now realize this occured in the field on a version that did not have that option yet. Though I'd be pleasantly surprised if it works "AMVAA" is one character less than "ASCII ".
